modules/gcp_nodepool/gcp-nodepool.yaml

```diff
@@ -70,6 +70,11 @@ inputs:
     validator: list(include('taint'), required=False)
     description: Taints to add to the nodes in this nodegroup.
     default: []
+  - name: guest_accelerators
+    user_facing: true
+    validator: list(include('guest_accelerator'), required=False)
+    description: GPU accelerators to add to the nodes in this nodegroup.
```

Comment:
Will you add some commentary here about installing gpu drivers by default (or if not here, in the `use_latest_gpu_drivers` field)?

Reply:
To clarify, GPU drivers are always installed. The `use_latest_gpu_drivers` flag simply installs the latest Nvidia drivers as opposed to the default ones. I'll add a note about that. See: https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#installing_drivers

Reply:
Yep, initially misread which is why I was asking for clarification that they are installed by default :)